presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergNativeMetadata.java

```diff
@@ -166,8 +165,6 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
 
         Schema schema = toIcebergSchema(tableMetadata.getColumns());
 
-        verifyTypeSupported(schema);
```

Comment:
Same question

Reply:
I deleted this method, it only served to filter out the timestamp type.

Reply:
Hmmm. We also currently don't support the UUID type. I guess it fails in a different way then

Reply:
I think UUID is failing in the type converter default case